doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -172,14 +172,16 @@ include::../api/protos/vkCreateInstance.txt[]
   * pname:pInstance points a sname:VkInstance handle in which the resulting
     instance is returned.
 
-fname:vkCreateInstance creates the instance, then enables and initializes
-global layers and extensions requested by the application.
-If an extension is provided by a layer, both the layer and extension must:
-be specified at fname:vkCreateInstance time.
-If a specified layer cannot be found, no sname:VkInstance will be created
-and the function will return ename:VK_ERROR_LAYER_NOT_PRESENT.
-Likewise, if a specified extension cannot be found the call will return
+fname:vkCreateInstance verifies that the requested layers exist. If not,
+fname:vkCreateInstance will return ename:VK_ERROR_LAYER_NOT_PRESENT.
+Next fname:vkCreateInstance verifies that the requested extensions are
+supported (e.g. in implementation or enabled instance layers) and if
+any requested extension is not supported, fname:vkCreateInstance must: return
 ename:VK_ERROR_EXTENSION_NOT_PRESENT.
+After verifying and enabling the instance layers and extensions the
+sname:VkInstance object is created and returned to the application.
+If a requested extension is only supported by a layer, both the layer and
+extension need to be specified at fname:vkCreateInstance time.
```

Comment:
Would add " , for the creation to succeed." at the end to emphasize this is not a requirement. (Same for the `vkCreateDevice` copy)

Should be "and **the** extension"?